src/main/kotlin/DetektDemo.kt

```diff
@@ -1 +1,51 @@
-class DetektDemo {}
+class DetektDemo {
+    private   val unusedVariable :   String = "unused"
+
+    fun isHTTPSuccess(statusCode: Int): Boolean = statusCode == 200
+
+    fun veryLongFunction(){
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello");
+        println("Hello")
+        println("Hello")
+        println("Hello")
+        println("Hello");
+        println("Hello")
+        println("Hello")
+    }
+
+    fun fixMyStyle(x:Int)   : Int {
+        when(x){
+            1 ->{
+                println(x)
+                return x
+            }
+            2 ->{
+                println(x)
+                return x
+            }
+            else -> {throw Exception()}
+        }
+    }
+
+    fun notCompletedFunction() {
+        // TODO: fix this function
+        println("not completed")
+    }
+
+    // This is a tmp hack solution
+    fun hackyFunction(){
+        println("hack solution")
+    }
+
+    fun unsafeFunction(){
+        var user: User? = null
+        println(user!!.name)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/b1db8249b3b7765813bc2400c54d7c55f4c41ab9/src/main/kotlin/DetektDemo.kt#L45"></span>Calling !! on a nullable type will throw a NullPointerException at runtime in case the value is null. It should be avoided.</td>
    </tr>
  </tbody>
</table>